triplea_maps.yaml

```diff
@@ -938,28 +938,29 @@
   description: |
     <br>This is a MAP SKIN for "Diplomacy"
     <br>A very basic style.  I made this one for people who hate the cursive writing of the other 3 skins.  It has a simple font, no textures, and no cursive.
-  version: 1
+  version: 1.7
```

Comment:
You updated the wrong version number. This is the version for diplomacy. We are also just incrementing the version number in this file as integer numbers as its the map zip version not the individual maps.